modules/data-loading/partials/kafka/kafka-example-loading-job.adoc

```diff
@@ -2,7 +2,7 @@
 
 The following is an example loading job from and external Kafka cluster.
 
-[source,php,linenums]
+[source,sql,linenums]
```

Comment:
Let's use `gsql`

Reply:
Did you forget this one?

Reply:
Interesting, you didn't see `gsql`? From my local I am seeing `gsql` instead of `sql`.